### DISCUSSION TITLE
How do we most efficiently structure work to take advantage of everyone's skills?

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#477] [Ideas]
- Separating out C++, R, and pilot study work
- Clearer time allocation of big group meetings to things that need a decision/feedback vs process discussions that could proceed with a smaller group

**Comment by kellijohnson-NOAA** [comment]
This feedback from Christine almost two years ago is still relevant today and a good reminder that group meetings need set decision/feedback processes. I encourage anyone who has ideas on how to get more out of group meetings in FIMS to add comments to this Discussion. Also, documenting pain points without any idea regarding how to fix it also warrants a comment.
